pkg/persistence/postgres/table_version.go

```diff
@@ -0,0 +1,32 @@
+package postgres // import "github.com/joincivil/civil-events-crawler/pkg/persistence/postgres"
+
+import (
+	"fmt"
+	// "github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	versionTableName = "version"
+)
+
+// CreateVersionTableQuery returns the query to create the version table
+func CreateVersionTableQuery() string {
+	return CreateVersionTableQueryString(versionTableName)
+}
+
+// CreateVersionTableQueryString returns the query to create this table
+func CreateVersionTableQueryString(tableName string) string {
+	queryString := fmt.Sprintf(`
+        CREATE TABLE IF NOT EXISTS %s(
+            version TEXT,
+            service_name TEXT
+        );
+    `, versionTableName)
+	return queryString
+}
+
+// VersionData is the model for the version table in DB
+type VersionData struct {
```

Comment:
Can prob just call this `Version`

Reply:
haha, but it's not ok for the table name to just be `version`?

Reply:
lol, i was going back and forth on the naming myself.  just making sure it all make sense is all.  if you don't think what i'm saying makes sense, feel free to choose.